public/app/features/transformers/docs/content.ts

```diff
@@ -1206,28 +1320,50 @@ export const transformationDocsContent: TransformationDocsContentType = {
   `;
     },
   },
   spatial: {
     name: 'Spatial',
     getHelperDocs: function () {
-      // This template string space-formatting is intentional.
       return `
-  Use this transformation to apply spatial operations to query results
+  Use this transformation to apply spatial operations to query results.
+  
+  - **Action** - Select an action:
+    - **Prepare spatial field** - Set a geometry field based on the results of other fields.
+      - **Location mode** - Select a location mode (these options are shared in **Calculate value** and **Transform** modes):
+        - **Auto** - Automatically identify location data based on default field names.
+        - **Coords** - Specify latitude and longitude fields.
+        - **Geohash** - Specify a geohash field.
+        - **Lookup** - Specify Gazetteer location fields.
+    - **Calculate value** - Use the geometry to define a new field (heading/distance/area).
+      - **Function** - Choose a mathematical operation to apply to the geometry:
+        - **Heading** - Calculate the heading (direction) between two points.
+        - **Area** - Calculate the area enclosed by a polygon defined by the geometry.
+        - **Distance** - Calculate the distance between two points.
+    - **Transform** - Apply spatial operations to the geometry.
+      - **Operation** - Choose an operation to apply to the geometry:
+        - **As line** - Create a single line feature with a vertex at each row.
+        - **Line builder** - Create a line between two points.
+  
+  This transformation allows you to manipulate and analyze geospatial data, enabling operations such as creating lines between points, calculating spatial properties, and more.
   `;
     },
   },
   timeSeriesTable: {
     name: 'Time series to table transform',
     getHelperDocs: function () {
       return `
-  Use this transformation to convert time series result into a table, converting time series data frame into a "Trend" field. "Trend" field can then be rendered using [sparkline cell type][], producing an inline sparkline for each table row. If there are multiple time series queries, each will result in a separate table data frame. These can be joined using join or merge transforms to produce a single table with multiple sparklines per row.
+  Use this transformation to convert time series results into a table, transforming a time series data frame into a "Trend" field. The "Trend" field can then be rendered using the [sparkline cell type][], generating an inline sparkline for each table row. If there are multiple time series queries, each will result in a separate table data frame. These can be joined using join or merge transforms to produce a single table with multiple sparklines per row.
```

Comment:
Assuming the Trend field is something in the UI, I've formatted it as such
```suggestion
  Use this transformation to convert time series results into a table, transforming a time series data frame into a **Trend** field. The **Trend** field can then be rendered using the [sparkline cell type][], generating an inline sparkline for each table row. If there are multiple time series queries, each will result in a separate table data frame. These can be joined using join or merge transforms to produce a single table with multiple sparklines per row.
```